vcpkg/concepts/asset-caching.md

```diff
@@ -28,15 +34,19 @@ Asset caching is configured via:
 * The `X_VCPKG_ASSET_SOURCES` environment variable, or
 * The `--x-asset-sources` command-line option.
 
-In both cases, the expected value is a semicolon-delimited list of sources. Each
-source has a specific syntax depending on its storage backend (see the [asset
-caching reference documentation](../users/assetcaching.md) to learn how to
-configure asset caching sources).
+In both cases, the expected value is a semicolon-delimited list of sources. Each source has a specific syntax depending
+on its storage backend. See the [asset caching reference documentation](../users/assetcaching.md) to learn how to
+configure asset caching sources.
+
+### Example using Azure Blob Storage
+
+```PowerShell
+vcpkg install zlib --x-asset-sources="clear;x-azurl,mystorageaccount.blob.core.windows.net,${env:SAS_TOKEN},readwrite"
```

Comment:
1. I think we should show a normal variable like `$SAS_TOKEN` rather than an `$env` one. In particular, I think we should discourage putting secrets into env vars when we can do so.
2. Given that this is documenting semicolon delimited, we might want an example with more than one? I know `clear` is here but as it has no `,` delimited parts it might not clarify much.

Reply:
```suggestion
vcpkg install zlib --x-asset-sources="clear;x-azurl,mystorageaccount.blob.core.windows.net,${SAS_TOKEN},readwrite;x-azurl,file:///Z:/vcpkg-cache/assets,,readwrite"
```